### PR TITLE
[Dashboard] Add memory unit converter test

### PR DESCRIPTION
**Pull request by Linniem**
## Why are these changes needed?

Add missing test(#30508).

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by Linniem** [comment]
@alanwguo please help review, thank you.
cc @rkooo567

**Comment by rkooo567** [comment]
Screen Shot 2023-03-16 at 6 59 45 AM
this PR also breaks the master dashboard tests. So we may need to revert it. Or do you think you can handle this quickly from your PR here? (the policy is to revert asap, but if you can make a fix quickly, I think we can just merge this PR)

**Comment by Linniem** [comment]
@rkooo567 please help revert #30508.
I need time to check it. Thank you.

**Comment by ollie-iterators** [comment]
I believe that there are ways to make it so that commits are not merged if certain tests do not pass

File: dashboard/client/src/util/converter.unit.test.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      input: 9 * 1024 ** 5,
+      expected: "9.00PB",
+    },
+  ];
+
+  tests.forEach((test) => {
+    it(test.name, () => {
+      expect(memoryConverter(test.input)).toEqual(test.expected);
+    });
+  });
```
Comment (alanwguo) [inline dashboard/client/src/util/converter.unit.test.ts:41]: Can you change the syntax slightly to use `it.each`?
https://jestjs.io/docs/api#testeachtablename-fn-timeout

**Review by alanwguo — APPROVED** [review]
Thanks!

**Comment by Linniem** [comment]
Remove commit that fix failed tests in this pull request, which already done with #33377.

**Comment by Linniem** [comment]
@alanwguo  Could you merge the pull request? Thank you.

**Pull request merged**